projects/gen-ai.md

```diff
@@ -0,0 +1,157 @@
+
+By providing GenAI community with the space under OpenTelemetry we will be able to start hosting vendor-agnostic instrumentation libraries (such as WIP one [OpenAI Python](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2759)) and eventually we will be able to move popular OpenLLMetry-owned `opentelemetry-instrumentation-*` libraries to OTel (or to upstream libraries).
+
+It helps GenAI observability community to
+
+- find solutions for new observability problems such as verbose events or correlation with async response evaluations
+- discuss implementation challenges
+- coordinate and prioritize common efforts
+
+### Goals, objectives, and requirements
+
+GenAI observability project is a new OTel cross-cutting SIG that intends to provide the GenAI community with a space to evolve and maintain GenAI related instrumentations in a **vendor-agnostic way**.
+
+It includes:
+
+- LLM Semantic Conventions
+- Instrumentations for GenAI client libraries, frameworks, or other components.
+
+The eventual goal is to contribute instrumentation libraries to upstream components or to otel-contrib repos when upstream contributions are not possible.
+
+During initial development and experimentation, we may need to release libraries more frequently than it's possible for contrib and then may temporarily host instrumentation libraries in a `opentelemetry-{language}-genai` repository(ies).
+
+This would help GenAI observability vendors that will depend on these libraries to patch issues and adapt to external changes.
+
+## Deliverables
+
+Immediate term:
+- Ship OTel instrumentation libraries for OpenAI (or any other GenAI client) in Python and JS following exiting conventions
+
+Middle term:
+- Ship OpenTelemetry (or native) instrumentations for popular GenAI client libraries in Python and JS covering chat calls
+- Evolve GenAI semantic conventions to cover other popular GenAI operations such as embeddings, image or audio generation
+
+As a result we should have feature parity with OpenLLMetry for a set of client instrumentation libraries that all vendors can depends upon.
+
+Long term:
+- Implement instrumentations for GenAI orchestrators and LLM frameworks for popular libraries in different language
+- Evolve GenAI and other relevant conventions (DB) to cover complex multi-step scenarios such as RAG
+- Propose mature instrumentations to upstream libraries/frameworks
+
+## Staffing / Help Wanted
+
+The initial set of approvers for each language must include:
+
+- representatives of at least 2 different observability vendors that are active members of OTel LLM SemConv WG
+- at least one existing approver or maintainer of the contrib, instrumentation or core OTel repo in the corresponding language
+
+**Project Leads:**
+
+- @drewby
+- @nirga
+
+**Sponsoring Members:**
+
+- @lmolkova
+- TBD
+
+**Engineers:**
+- Python
+  - maintainers/approvers:
```

Comment:
in otel terminology we have both maintainer and approver defined, are we saying in these repos there is no difference? Cool with me, just curious.

Reply:
we don't usually separate maintainers and approvers in project templates.

For this project I believe we'll work in the following repos:
- central repo for issues/discussions/maybe prototypes/artifacts, but probably no implementation code - `opentelmetry-genai` (new). 
  - Maintainers/approvers listed here will be the same for this repo
- [semantic-conventions](https://github.com/open-telemetry/semantic-conventions) and corresponding [semconv-llm-approvers](https://github.com/orgs/open-telemetry/teams/semconv-llm-approvers) which are codeowners for GenAI files. I think we need to expand this group and include more people who've been active and consistent contributors to semantic conventions.
- [opentelemetry-js-contrib](https://github.com/open-telemetry/opentelemetry-js-contrib) maintainers/approvers for JS will become [component owners](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/CONTRIBUTING.md#component-ownership), I hope @trentm would keep me honest here
- [opentelemtery-python-contrib](https://github.com/open-telemetry/opentelemetry-python-contrib) -similar [component-ownership](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/.github/component_owners.yml) model
- new repos - it seems that we would need to start python work in a separate repo and if it's the case, we'll need to decide. I updated the list and added existing Otel Python/JS maintainers to maintainers for potential new repos.